src/makeCalculator.js

```diff
@@ -38,6 +38,29 @@
  */
 function makeCalculator() {
   // write code here
+  let result = {
+    result: 0,
+    add: (value) => {
+      this.result += value;
+    },
+    subtract: (value) => {
+      this.result -= value;
+    },
+    multiply: (value) => {
+      this.result *= value;
+    },
```

Comment:
Use non-arrow functions for object methods (because  arrow function takes `this` from scope)

Please rewrite in this way:
![image](https://github.com/mate-academy/js_advanced_calculator/assets/91946036/eaaad01f-b33b-4cc6-afed-a3536e15ede1)

This is a function that will  always receive meaningful `this` value from their caller